packages/esm-patient-forms-app/src/hooks/use-forms.ts

```diff
@@ -19,22 +19,28 @@ import {
 } from '../constants';
 import { isValidOfflineFormEncounter } from '../offline-forms/offline-form-helpers';
 
-export function useFormEncounters(cachedOfflineFormsOnly = false, patientUuid: string = '', visitUuid: string = '') {
+function useCustomFormsUrl(patientUuid: string, visitUuid: string) {
   const { customFormsUrl, showHtmlFormEntryForms } = useConfig<ConfigObject>();
   const hasCustomFormsUrl = Boolean(customFormsUrl);
-  let url = hasCustomFormsUrl
-    ? customFormsUrl.indexOf('?') === -1
-      ? `${customFormsUrl}?patientUuid=${patientUuid}`
-      : customFormsUrl
-    : showHtmlFormEntryForms
-      ? formEncounterUrl
-      : formEncounterUrlPoc;
-  url = interpolateUrl(url, {
+
+  let baseUrl = hasCustomFormsUrl ? customFormsUrl : showHtmlFormEntryForms ? formEncounterUrl : formEncounterUrlPoc;
```

Comment:
Shouldn't `baseUrl` be a `const` now?

Does anyone have any clue what this whole `formEncounterUrlPoc` thing is, and whether we need to support it still?  What's the deal here?

Reply:
If i recall this correctly, `formEncounterUrlPoc` was added to support Ampath forms, The Angular form engine developed by Ampath had a naming convention, where `POC` was prefix to actual form and in cases the form name missed the prefix, it was treated as a reference form, this enabled them to only fetch correct forms for users to work with.

Reply:
I think we can retired `formEncounterUrlPoc` and have implementations use custom url e.g `/mymodule/patientforms?patient=${patientUuid}&visit=${visitUuid}&q=poc` cc @Alfred-Mutai @Kipkemoii